tool/resources/org/antlr/v4/tool/templates/codegen/Python3/Python3.stg

```diff
@@ -561,7 +561,7 @@ TokenPropertyRef_text(t) ::= "(None if <ctx(t)>.<t.label> is None else <ctx(t)>.
 TokenPropertyRef_type(t) ::= "(0 if <ctx(t)>.<t.label> is None else <ctx(t)>.<t.label>.type()"
 TokenPropertyRef_line(t) ::= "(0 if <ctx(t)>.<t.label> is None else <ctx(t)>.<t.label>.line)"
 TokenPropertyRef_pos(t) ::= "(0 if <ctx(t)>.<t.label> is None else <ctx(t)>.<t.label>.column)"
-TokenPropertyRef_channel(t) ::= "(0 if (<ctx(t)>.<t.label> is None else <ctx(t)>.<t.label>.channel)"
+TokenPropertyRef_channel(t) ::= "(0 if <ctx(t)>.<t.label> is None else <ctx(t)>.<t.label>.channel)"
```

Comment:
I guess the same issue should be fixed in Python2.stg too: https://github.com/antlr/antlr4/blob/4.7.1/tool/resources/org/antlr/v4/tool/templates/codegen/Python2/Python2.stg#L555